spec/conman_spec.rb

```diff
@@ -36,24 +36,49 @@
     expect_field(:notes, "notes")
   end
 
-  it "adds two contacts through the main loop" do
+  it "adds two contacts through the add-contacts loop" do
```

Comment:
"Loop" is a bit too technical when it comes to test descriptions.
What's the behaviour the loop exposes? Contacts are added until the user choses to be done with it, aren't they?
The test itself is not wrong, but I want to focus on the documentation side of this test.